src/main/java/org/skriptlang/skript/bukkit/registration/BukkitSyntaxInfosImpl.java

```diff
@@ -27,7 +27,7 @@ static final class EventImpl<E extends SkriptEvent> implements Event<E> {
 		private final ListeningBehavior listeningBehavior;
 		private final String name;
 		private final String id;
-		private final @Nullable String since;
+		private final Collection<String> since;
 		private final @Nullable String documentationId;
```

Comment:
```suggestion
		private final @Nullable String documentationId;
		private final Collection<String> since;
```
Just so the ordering is consistent with the methods